src/ua/hillel/Cat.java

```diff
@@ -0,0 +1,24 @@
+public class Cat extends Animals {
+    int max_run = 200;
```

Comment:
The same here, please follow the naming conventions. In Java, we use Camel case to name our methods, classes, and variables, so the underscore "_" must not be used. For example, `max_run` should be renamed to `maxRun` etc. Please rename all variables in this class and make them private.